platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_platform.sh

```diff
@@ -191,6 +195,28 @@ xcvr_presence_interrupts() {
     esac
 }
 
+# Reset the mux tree
+reset_muxes() {
```

Comment:
for resetting the muxes, is it possible to reset only one mux per the failure? or do we need to reset all the muxes? if we reset all the muxes, do we need to re-build the whole i2c tree?

Reply:
It is possible to reset only the specific mux that are part of this channel - however the complexity of that logic is not warranted. Resetting the muxes does not necessitate rebuilding the kernel's i2c tree.